fluidefi/fluidefi-caspernet-aggregator-tools
# Deploy fetcher

**Summary of change:**
- Added a new micro-service, deployFetcher, which uses the casper sdk method getDeployInfo().

**List of tasks:**
- [x] Defined types.
- [x] Defined interfaces.
- [x] Defined class.
- [x] Created unit test.
- [x] Generated the docs. 

**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/deploy-fetcher`
```bash
git checkout feature/deploy-fetcher
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm deployFetcher test has passed.
- Run the docs using live server and make sure it have been updated.
